From 251684169627f0aae205c9a316863e0e3f257912 Mon Sep 17 00:00:00 2001
From: John Khvatov <ivaxer@fedoraproject.org>
Date: Thu, 1 May 2014 00:57:04 +0400
Subject: [PATCH] cmake: append LIB_SUFFIX to LIBRARY_INSTALL_DIR

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 17d38b4..10c9cca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -280,9 +280,9 @@ if (OS_WINDOWS)
   set (CONFIG_INSTALL_DIR  CMake)
 else ()
   set (RUNTIME_INSTALL_DIR bin)
-  set (LIBRARY_INSTALL_DIR lib)
+  set (LIBRARY_INSTALL_DIR lib${LIB_SUFFIX})
   set (INCLUDE_INSTALL_DIR include)
-  set (CONFIG_INSTALL_DIR  lib/cmake/${PACKAGE_NAME})
+  set (CONFIG_INSTALL_DIR  lib${LIB_SUFFIX}/cmake/${PACKAGE_NAME})
 endif ()
 
 file (RELATIVE_PATH INSTALL_PREFIX_REL2CONFIG_DIR "${CMAKE_INSTALL_PREFIX}/${CONFIG_INSTALL_DIR}" "${CMAKE_INSTALL_PREFIX}")
-- 
1.9.0

